tests/schema/types/test_date.py

```diff
@@ -0,0 +1,138 @@
+import datetime
+
+import pytest
+
+from graphql import GraphQLError
+
+import strawberry
+
+
+@pytest.mark.parametrize(
+    "typing,instance,serialized",
+    [
+        (datetime.date, datetime.date(2019, 10, 25), "2019-10-25"),
+    ],
+)
+def test_serialization(typing, instance, serialized):
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        def serialize(self) -> typing:
+            return instance
+
+    schema = strawberry.Schema(Query)
+
+    result = schema.execute_sync("{ serialize }")
+
+    assert not result.errors
+    assert result.data["serialize"] == serialized
+
+
+@pytest.mark.parametrize(
+    "typing,name,instance,serialized",
+    [
+        (datetime.date, "Date", datetime.date(2019, 10, 25), "2019-10-25"),
```

Comment:
Is it likely that we will add more parameters in the future? Otherwise, it might be better to keep it simple and inline the parameters to reduce complexity and increase readability. Any thoughts?

Reply:
@DoctorJohn Just to confirm, would you prefer me to change it?

Reply:
Personally, I'd prefer it.

Reply:
Thanks @joeydebreuk